### ISSUE TITLE
2.3: CommandStation EX fails with Ethernet.h error

### ISSUE THREAD

**Issue by ggee** [DCC-EX/exInstaller#12]
CommandStation EX fails to compile.  Failure comping Ethernet support, probably because it didn't download the library??

Install options chosen, CommandStation EX, Mega, Arduino Motor Shield, COM7, WIfi, plus wifi settings and Windows 10 using x64 installer.

```
Compiling CommandStation EX Sketch
Using board 'mega' from platform in folder: C:\Users\ggee\Downloads\dccex\Installer-win-x64\packages\arduino\hardware\avr\1.8.3
Using core 'arduino' from platform in folder: C:\Users\ggee\Downloads\dccex\Installer-win-x64\packages\arduino\hardware\avr\1.8.3
Detecting libraries used...
"C:\Users\ggee\Downloads\dccex\Installer-win-x64\packages\arduino\tools\avr-gcc\7.3.0-atmel3.6.1-arduino7/bin/avr-g++" -c -g -Os -w -std=gnu++11 -fpermissive -fno-exceptions -ffunction-sections -fdata-sections -fno-threadsafe-statics -Wno-error=narrowing -flto -w -x c++ -E -CC -mmcu=atmega2560 -DF_CPU=16000000L -DARDUINO=10607 -DARDUINO_AVR_MEGA2560 -DARDUINO_ARCH_AVR "-IC:\Users\ggee\Downloads\dccex\Installer-win-x64\packages\arduino\hardware\avr\1.8.3\cores\arduino" "-IC:\Users\ggee\Downloads\dccex\Installer-win-x64\packages\arduino\hardware\avr\1.8.3\variants\mega" "C:\Users\ggee\AppData\Local\Temp\arduino-sketch-FD3334BFC5508AE676C927A7FB96E17D\sketch\CommandStation-EX.ino.cpp" -o nul -DARDUINO_LIB_DISCOVERY_PHASE
Alternatives for Ethernet.h: []
ResolveLibrary(Ethernet.h)
  -> candidates: []
Failed to compile sketch in ./CommandStation-EX/. got error exit status 1
Compile failed please double check sketch and try again

```

**Comment by dexslab** [comment]
Library is downloaded and is proper one just not sure why its not being detected or something else

**Comment by ggee** [comment]
This seems to work in your latest patch zip you posted.

**Comment by ggee** [comment]
Fixed in #13
